### ISSUE TITLE
Pin issue about contribution procedure

### ISSUE THREAD

**Issue by lasjorg** [freeCodeCamp/freeCodeCamp#52056]
### Describe the Issue

Instead of us having to constantly tell people how we handle issues, when a PR should be opened, where the contribution documentation is, etc. can we try pinning a "READ ME FIRST" issue that briefly explains the procedure and links to the contribution documentation?

I know it won't fix the problem completely but it might catch the attention of enough people to cut back on some of the boilerplate responses we have to make.

Anyway, it's just an idea.

### Affected Page

https://github.com/freeCodeCamp/freeCodeCamp/issues

### Your code

```



```

### Expected behavior

Fewer boilerplate responses on issues.

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

**Comment by ieahleen** [comment]
rule 1: people don't read, otherwise we would have more people reading the contributing documentation, which is linked in the README...
But we could try

**Comment by ShaunSHamilton** [comment]
In my opinion, I cannot see a pinned issue catching any more people than those who were already going to read the contributing docs anyway. _We can always try it, though_
I suspect, the only way to truly catch more people would be for GitHub to have a feature whereby clicking on an issue redirected you to a page requiring action (e.g. "check box to confirm you have read above guidelines") to be redirected back to the issue.

**Comment by moT01** [comment]
I agree that most people probably won't notice or read the pinned issue - but I kind of like it as something to reference. Instead of having to write a message in every issue or PR explaining everything - we can just link to the pinned issue.

**Comment by lasjorg** [comment]
I'd also like to point out that a pinned issue will function much like a footer on a website, no matter how you get to the current page you are on, general information is always available in the footer. The assumption that the order of events will be main repo > readme > contribution docs > issue tracker, while logical is in no way guaranteed.

As mentioned the pinned issue can serve as a nice reference as well.
